src/components/Menu/LanguageNavigation/LanguageNavigation.tsx

```diff
@@ -89,7 +94,7 @@ const LanguageNavigation = ({
       ) : null}
 
       {items.length >= 1 ? (
-        <div className="border-b border-charcoal-grey w-full">
+        <div className="w-full border-b border-charcoal-grey">
```

Comment:
Was this re-ordering performed by some sort of linter?

Reply:
I believe it was performed by some sort of Peter :smile: 
Jokes aside, he did do it on a lot of places in this pr, but I thought I caught them all